engine/src/main/java/io/zeebe/engine/processing/common/EventTriggerBehavior.java

```diff
@@ -342,4 +254,25 @@ public void activateTriggeredEvent(
     commandWriter.appendFollowUpCommand(
         eventInstanceKey, ProcessInstanceIntent.COMPLETE_ELEMENT, eventRecord);
   }
+
+  private void activateEventSubProcess(
+      final ExecutableFlowElement triggeredEvent, final ExecutableFlowElement flowScope) {
+    // first activate the event sub process
```

Comment:
Lets improve the comment such that we understand also later why we not write the command here and immediately go to ACTIVATED and add that we write the ACTIVATE command for the start event.

Reply:
I can also do it, its just a reminder.

Reply:
Sure, that makes total sense